Makefile

```diff
@@ -35,13 +35,14 @@ MOCK_CONFIG=default
 all:
 	@echo
 	@echo "Development related targets:"
-	@echo "check:      Runs tree static check, unittests and fast functional tests"
-	@echo "check-full: Runs tree static check, unittests and all functional tests"
-	@echo "develop:    Runs 'python setup.py --develop on this tree alone"
-	@echo "link:       Runs 'python setup.py --develop' in all subprojects and links the needed resources"
-	@echo "clean:      Get rid of scratch, byte files and removes the links to other subprojects"
-	@echo "selfcheck:  Runs tree static check, unittests and functional tests using Avocado itself"
-	@echo "spell:      Runs spell checker on comments and docstrings (requires python-enchant)"
+	@echo "check:          Runs tree static check, unittests and fast functional tests"
+	@echo "check-extended: Runs tree static check, unittests and most functional tests"
+	@echo "check-full:     Runs tree static check, and all unittests and functional tests"
```

Comment:
Once again, do you think people will use all of these regularly? We already have a lot of target I don't see a reason to expand it. I'd suggest:

```
check:       Runs default set of static checks, unittests and some functional tests (Level can be tweaked by AVOCADO_CHECK_LEVEL=0-2)
check-full: Runs all static checks, unittests and functional tests
```

Anyway it's a minor thing so please just let me know which one you prefer.